src/main/java/pl/mmieczak/springbootexercise_22_1/UserController.java

```diff
@@ -0,0 +1,48 @@
+package pl.mmieczak.springbootexercise_22_1;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Controller
+public class UserController {
+
+    private pl.mmieczak.springbootexercise_22_1.UserRepository userRepository;
```

Comment:
Nie dawaj `full qualified name` (czyli pełnej nazwy klasy z pakietami). Staraj się ograniczać takie użycia.

Reply:
kopiowałem kod bo mialem problem z gitem i chciałem bulla przygotować i nie zauważyłem ze InteliJ chyba sam mi to rozwinął w ten sposób, nie wpisałem tego ręcznie